HDF5Examples/config/cmake/jrunTest.cmake

```diff
@@ -102,10 +102,14 @@ endif ()
 if (TEST_MASK_ERROR)
   if (NOT TEST_ERRREF)
     # the error stack has been appended to the output file
-    file (READ ${TEST_FOLDER}/${TEST_OUTPUT} TEST_STREAM)
+    if (EXISTS "${TEST_FOLDER}/${TEST_OUTPUT}")
```

Comment:
Thinking about this more, would it be better to just exit/return an error/etc. when the test file doesn't exist? Otherwise, it seems like `TEST_STREAM` could be unset which might cause another problem further on.

Reply:
In most of these files it looks like `TEST_STREAM` isn't particularly local since it's just set inside an if block and used outside those blocks later on. For example, it's printed out just a bit below this line which won't particularly do anything when it's empty, but also wouldn't be very helpful. It seems like a good argument for use of functions if we want to restrict scope of certain variables (which I'm all for).

Reply:
> In most of these files it looks like `TEST_STREAM` isn't particularly local

Reviewing the files again.